Component/Badge/badge.css

```diff
@@ -0,0 +1,53 @@
+@import url(/CSS/docs.css);
+@import url(/Component/Avatar/avatar.css);
+
+.text{
+    position: relative;
+    display: inline-flex;
+}
+.badge{
+    position: relative;
+}
+.badge-icon{
+    color: white;
+    position: absolute;
+    top: -1rem;
+    right: -1rem;
+    padding: 0.5rem 0.75rem;
+    height: 1.5rem;
+    width: 1rem;
+    background-color: var(--blue);
+    border-radius: 50%;
+    line-height: 1.5rem;
+    font-family: 'Mate', serif;
+    font-size: 1.5rem;
+}
+
+
+button{
+    padding: 0.75rem 1rem;
+}
```

Comment:
I think it's not preferred to apply style directly to the HTML element. Since, when you'll import these stylesheets it might get applied to all your buttons.

Reply:
ya since I haven't designed the button component yet. Hence I applied it directly. And yes as you said when ill import buttons.css it will be removed from main.css